pages/_app.js

```diff
@@ -86,6 +91,41 @@ export default function App({ Component, pageProps }) {
     showToastMessage("Successfully edited!");
   }
 
+  const existingEmotionTypes = [
+    "Love",
+    "Tired",
+    "Annoyed",
+    "Proud",
+    "Hopeful",
+    "Confident",
+    "Excited",
+    "Releived",
+  ];
```

Comment:
this information is already present in your emotionTypes state. Is this list used to determine unused emotions or is it the entire list of possible emotions?

Reply:
now unnecessary and removed